index.html

```diff
@@ -38,6 +38,22 @@ <h2>Connect</h2>
       <a href="mailto:nn.dewi.anggraini@gmail.com">Dewi-Anggraini</a>
     </section>
     <img src="https://shop.static.ingka.ikea.com/category-images/Category_dining-sets.jpg">
+    <section>
+      <h2>Leave a Message</h2>
+      <form name="leave_message">
+        <label for="users_name">Users Name</label><br>
+        <input type="text" name="usersName" required="true"><br>
+        <label for="users_email">Users Email</label><br>
+        <input type="text" name="usersEmail" required="true"><br>
+        <label for="users_message">Users Message</label><br>
+        <textarea name="usersMessage" required="true"></textarea><br>
```

Comment:
The input and textarea elements should have an id matching the 'for' attribute of the label element.